packages/admin-ui/src/components/Sidebar/__stories__/Sidebar.stories.tsx

```diff
@@ -147,29 +154,41 @@ export const Playground: Story<any> = (args) => {
   return (
     <Box
       csx={{
-        height: '100%',
-        width: '100%',
-        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        maxHeight: '100vh',
       }}
+      data-testid="container-shell"
     >
-      <Box
-        csx={{
-          height: 30,
-          padding: 22,
-          display: 'flex',
-          justifyContent: 'center',
-          alignItems: 'center',
-          border: '4px dashed black',
-          margin: 1,
-        }}
-      >
-        TOP BAR
-      </Box>
+      <Topbar>
+        <TopbarStart>
+          <Set spacing={3}>
```

Comment:
Do spacing tokens work here?

Reply:
Not yet, we should fix it once we review this component